required/tools/testfiles/github-1765.lvt

```diff
@@ -0,0 +1,41 @@
+\documentclass{article}
+
+\input{test2e}
+
+\begin{document}
```

Comment:
Need to load the `array` package here?

Also, I find this new test already gives expected output in LaTeX 2025-06-01, without the patch here. So what's the purpose of this test?

Reply:
The purpose is to test the correction made for the blunder in array introduced in 2025/06 with the wrong definition of \@finalstrut. Without array the old definition in ltboxes works as well (but doesn't if array is loaded). Of cuorse it helps to actually load array in the test then